src/utils/SortScenarioListUtils.js

```diff
@@ -27,3 +33,12 @@ export const sortScenarioList = (scenarioList) => {
 
   return sortedList;
 };
+
+export const getFirstScenarioMaster = (scenarioList) => {
+  if (scenarioList.length > 0) {
+    const scenarioMasterList = scenarioList.filter((scenario) => scenario.parentId === null);
+    return _sortByName(scenarioMasterList)[0];
```

Comment:
If the scenario list data is corrupted, it may contain only non-root scenarios (scenarios whose parent is not null). In this case, `scenarioMasterList` will be empty and reading its first element with `_sortByName(scenarioMasterList)[0]` will make the application crash.
Could we add the following safeguard please ?
```
const scenarioMasterList = scenarioList.filter((scenario) => scenario.parentId === null);
if (scenarioMasterList.length === 0) {
  console.warn('Scenario list is corrupted, no root scenarios found');
  return _sortByName(scenarioList)[0];
}
return _sortByName(scenarioMasterList)[0];
```